src/every.ts

```diff
@@ -0,0 +1,100 @@
+import type {
+  Pred,
+  PredIndexed,
+  PredIndexedOptional,
+  TypePred,
+  TypePredIndexed,
+} from "./_types";
+import { purry } from "./purry";
+
+/**
+ * Checks if every element of `data` passed the predicate `predicate`.
+ *
+ * @param data - The Array to check.
+ * @param predicate - A predicate function to run.
+ * @returns A `boolean` indicating if every element in `data` passes the predicate `pred`.
+ * @signature
+ *    R.every(arr, predicate)
+ * @example
+ *    R.every([1, 2, 3], R.isNumber) // => true
+ *    R.every(['a', 0, 'b'], R.isString) // => false
+ * @dataFirst
+ * @indexed
+ * @pipeable
```

Comment:
**issue:** i think we only use `@pipeable` for lazy functions, so we should remove this

Reply:
(or at least, we should)

Reply:
My first pr after releasing v2 would be to rename pipeable as lazy